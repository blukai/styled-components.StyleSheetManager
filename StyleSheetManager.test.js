import React from 'react';
import render from 'react-test-renderer';
import styled, { StyleSheetManager } from 'styled-components'
import 'jest-styled-components';

const Flower = styled.div`
  display: flex;
`;

it('creates snapshot of styles', () => {
  const tree = render.create(<Flower />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("doesn't create snapshot of styles", () => {
  const tree = render.create(
    <StyleSheetManager disableVendorPrefixes>
      <Flower />
    </StyleSheetManager>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
