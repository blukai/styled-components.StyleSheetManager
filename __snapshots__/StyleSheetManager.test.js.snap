// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates snapshot of styles 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0"
/>
`;

exports[`doesn't create snapshot of styles 1`] = `
<div
  className="sc-AykKC htpBwX"
/>
`;
